File: eclipse-plugins/edu.wpi.first.wpilib.plugins.java/resources/templates/examples/PacGoat/src/org/usfirst/frc/team190/pacgoat/commands/Collect.java
package $package.commands;

import $package.subsystems.Collector;
import $package.subsystems.Pivot;

import edu.wpi.first.wpilibj.command.CommandGroup;

/**
 * Get the robot set to collect balls.
 */
public class Collect extends CommandGroup {
    public Collect() {
    	addSequential(new SetCollectionSpeed(Collector.FORWARD));
    	addParallel(new CloseClaw());
    	addSequential(new SetPivotSetpoint(Pivot.COLLECT));
    	addSequential(new WaitForBall());
    }
}

File: eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/resources/templates/examples/PacGoat/src/Commands/Collect.cpp
#include "Collect.h"

#include "Robot.h"
#include "Commands/SetCollectionSpeed.h"
#include "Commands/CloseClaw.h"
#include "Commands/SetPivotSetpoint.h"
#include "Commands/WaitForBall.h"

Collect::Collect() {
	AddSequential(new SetCollectionSpeed(Collector::FORWARD));
	AddParallel(new CloseClaw());
	AddSequential(new SetPivotSetpoint(Pivot::COLLECT));
	AddSequential(new WaitForBall());
}
